pages/actual.py

```diff
@@ -0,0 +1,15 @@
+
+class Actual:
```

Comment:
Why are the file and class names actual?? The name should be descriptive of the file's content.